### ISSUE TITLE
Does not understand readiness_check_results

### ISSUE THREAD

**Issue by nhandler** [thefactory/marathon-python#94]
It looks like #92 added `readiness_checks` but not `readiness_check_results`

```
  File "/tmp/local/lib/python2.7/site-packages/marathon/client.py", line 579, in list_deployments
    return self._parse_response(response, MarathonDeployment, is_list=True)
  File "/tmp/local/lib/python2.7/site-packages/marathon/client.py", line 48, in _parse_response
    return [clazz.from_json(resource) for resource in target]
  File "/tmp/local/lib/python2.7/site-packages/marathon/models/base.py", line 35, in from_json
    return cls(**{to_snake_case(k): v for k, v in attributes.items()})
  File "/tmp/local/lib/python2.7/site-packages/marathon/models/deployment.py", line 28, in __init__
    for a in (current_actions or [])
  File "/tmp/local/lib/python2.7/site-packages/marathon/models/base.py", line 35, in from_json
    return cls(**{to_snake_case(k): v for k, v in attributes.items()})
TypeError: __init__() got an unexpected keyword argument 'readiness_check_results'
```

**Comment by nhandler** [comment]
@solarkennedy Can this be closed now that you've merged #95?

**Comment by solarkennedy** [comment]
Yes. Fixed in master.
